motr/idx.c

```diff
@@ -227,7 +227,7 @@ static int idx_op_init(struct m0_idx *idx, int opcode,
 		oi->oi_dtx = NULL;
 
 	if (opcode == M0_EO_CREATE && entity->en_type == M0_ET_IDX &&
-	    oi->oi_flags & M0_ENF_META) {
+	    entity->en_flags & M0_ENF_META) {
```

Comment:
better to use M0_IN instead of check for more readability

Reply:
But M0_IN() used for between two values of one variable. Here we are checking on value for opcode. Still i didn't get it. Can you explain more ?
Generally we are using M0_IN like
M0_IN(op->op_code, (M0_EO_CREATE, M0_EO_DELETE)

Reply:
Right, Lets go with existing method, we do not have any Macro to check for single parameter.